task_2/andrii.lukianchykov/task_2/task_2/task_2/src/main/java/com/bookshop/bookdatabase/BookDataBase.java

```diff
@@ -0,0 +1,85 @@
+package com.bookshop.bookdatabase;
+
+import com.bookshop.book.Book;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.System.out;
+
+public class BookDataBase {
+
+    private Book book;
+    private static List<Book> listOfBooks = new ArrayList<>();
```

Comment:
You are instantiating the list of books through a constructor. You don't need to declare it static.

Reply:
ok

Reply:
I have a problem with static here, if i delete "static" it has no access to get it(method) in outher classes